commcare_connect/static/sass/project.scss

```diff
@@ -94,3 +94,24 @@ $red: #b94a48;
   width: 100%;
   height: 2px;
 }
+
+table.table thead th.orderable a {
```

Comment:
Is this the same style that's used for sortable_link? If so, should we DRY it?

Reply:
This would require making changes to the sort_link implementation.